examples/paths/.swcrc

```diff
@@ -4,9 +4,11 @@
       "syntax": "typescript"
     },
     "target": "es2021",
-    "baseUrl": "examples/paths",
+    "baseUrl": "./src",
```

Comment:
did something change in swc where the contents of this file are now relative to the location of the file rather than the working directory?

Reply:
yeah I see the discussion on the PR thread about it